### ISSUE TITLE
About the VoVnet-99 checkpoint and config

### ISSUE THREAD

**Issue by zw615** [fundamentalvision/BEVFormer#133]
Hi, thanks for the great work!

I wonder if you could release the VoVnet-99 version checkpoint and config,  so that the test set results can be reproduced? Right now there is code for vovnet.py, but really no config or experiment use that code. 

Also, I wonder what pre-trained checkpoint for VoVnet-99 did you use? Was it first pre-trained on the COCO dataset for 2D Detection, and then pre-trained on the DDAM-15M dataset for depth prediction? Did you freeze any layer/stage in VoVnet-99 when training for 3D Detection?

Thanks a lot!

**Comment by wuhen777** [comment]
How did you replace ResNet-101 with VoVNet? I'm encountering some issues now
![image](https://github.com/fundamentalvision/BEVFormer/assets/84837506/5f4eb498-9012-4bdc-9c12-308f275aff9d)
This is the code I modified for bevformer_small.py
